test/check-application

```diff
@@ -51,13 +51,88 @@ class TestApplication(testlib.MachineCase):
         # show container listing toggle
```

Comment:
The "show all containers to interact with stopped ones" needs to be adjusted to the `podman run` changes above. Now `busybox` is the one that runs, and `alpine` the one that immediately stops. If that passes, it shows that this is missing a check that the status is actually shown as "stopped", and another check that it did *not* appear in the initial "running containers" list (which is why you have to select "All").